src/store/record.js

```diff
@@ -298,6 +300,39 @@ let recordStore = {
             let organisations = await client.executeQuery(recordOrganisationsQuery);
             commit('updateOrganisationsLinks', organisations.fairsharingRecord.organisationLinks);
         },
+        async updateAdditionalInformation({ state, commit}, options){
+            commit("resetMessage", "additionalInformation");
+            let access_points = state.sections.additionalInformation.data.access_points;
+            /*
+                What this does:
+                1. Get metadata (new GraphQL query)
+                2. Modify with access_points,
+                3. Post again (restClient.updateRecord).
+             */
```

Comment:
You don't need to do this. You can make a deep copy of state.generalInformation.data.metadata.
This will spare an extra not needed query. Make sure it's an hard copy though.